package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "nexus-search-webapp",
+  "version": "0.0.1",
+  "description": "Search web application for the Nexus Data Platform",
+  "main": "index.js",
+  "scripts": {
+    "start": "node index.js",
+    "test": "true"
```

Comment:
`true`? is that normal?

Reply:
"test" is mandatory in a package.json, for now there is no test, so I fall back on that. Needs to be replaced by real unit tests once there is real code in the proxy.